packages/block-editor/src/components/list-view/style.scss

```diff
@@ -553,13 +553,18 @@ svg {
 }
 
 .list-view-appender .block-editor-inserter__toggle {
-	background-color: #1e1e1e;
-	color: #fff;
-	margin: $grid-unit-10 0 0 24px;
-	height: 24px;
-	min-width: 24px;
+	background-color: $gray-900;
+	color: $white;
+	margin: $grid-unit-10 0 0 $grid-unit-30;
+	height: $button-size-small;
 	padding: 0;
 
+	// TODO: Consider passing size="small" to the Inserter toggle instead.
+	// Special dimensions for this button.
+	&.block-editor-inserter__toggle {
```

Comment:
Hmm, won't this result in a more specific selector, but with duplicate classname:

```
.list-view-appender .block-editor-inserter__toggle.block-editor-inserter__toggle
```

I'd expect a single use of the `.block-editor-inserter__toggle` classname, and the extra specificity could come from using the other classes (`.has-icon.is-next-40px-default-size`)

Reply:
I thought it would be better to avoid using class names derived from `components` packages as much as possible, just like class names with `.compnents-` prefixes. Is it okay to use `.has-icon` or `.is-next-40px-default-size`?

Reply:
I agree it's not ideal to use those classes, but if you don't use them at all, or just use one, precedence will likely not be enough.

Alternatively, you could always use `!important` and once the TODO to small size buttons intentionally is addressed, this will no longer be an issue.

Reply:
Updated by d6d9ad64cd27e0f666da756bade325567be3bfde👍

![image](https://github.com/user-attachments/assets/2b3edf77-2500-4fc2-9be7-ef7204c3d9ec)